### PR TITLE
CONTRIBUTING.es-final

### PR DESCRIPTION
**Pull request by haba-nero**
finalized Spanish translation for CONTRIBUTING FILE

### CODE REVIEW

**Comment by haba-nero** [comment]
This is my first pull request so please let me know if there is anything wrong with the files or if I did the request incorrectly.

**Review by fennifith — COMMENTED** [review]
This looks awesome - thank you for doing this! :D 

Left two small suggestions below!

It also seems like this might be based on a `CONTRIBUTING.md` from before we moved from unicorns.json into dedicated folders, so there may be some changes in the newest [CONTRIBUTING.md](https://github.com/unicorn-utterances/unicorn-utterances/blob/main/CONTRIBUTING.md) that didn't make it here. These are mostly small edits though - you can find the diff in [#867](https://github.com/unicorn-utterances/unicorn-utterances/pull/867/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055) (it may take a moment to load).

File: CONTRIBUTING.es.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+Cada código de idioma en [`/content/data/languages.json`](./content/data/languages.json) debe consistir de dos letras minúsculas. Si incluye una región, agregue un guion seguido de otras dos letras minúsculas. Por ejemplo, el código del idioma francés es `fr`; para especificar el dialecto francés de Canadá, el código sería `fr-ca`.
+
+>Por favor, use `-` en lugar de `_` en los formatos ISO de la región del idioma. En lugar de `fr_ca`, seria `fr-ca`.
+
+Consulte [Wikipedia: List of ISO 639-1 codes](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) para conocer los identificadores que se utilizaran en este formato.
+
+# Enviar una solicitud de extracción
+
+Una vez que todos los cambios se hayan hecho, [crea una solicitud de extracción](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) para unir tu publicación al sitio.
```
Comment (fennifith) [inline CONTRIBUTING.es.md:157]:
```suggestion
Una vez que todos los cambios se hayan hecho, [crea una solicitud de extracción](https://docs.github.com/es/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork) para unir tu publicación al sitio.
```
Just found that GitHub also has a Spanish version of this page that can be linked here :)

Comment (haba-nero) [reply CONTRIBUTING.es.md:157]: How do I implement these modifications? Do I have to do them manually or do I simply press the commit suggestion button?

Comment (fennifith) [reply CONTRIBUTING.es.md:157]: Yeah, the "commit suggestion" button will apply them!

The updates from [#867](https://github.com/unicorn-utterances/unicorn-utterances/pull/867/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055) would need to be done manually, though - it's mostly just a few sentences that were changed/added.

**Comment by haba-nero** [comment]
Oh ok, I have committed the two suggestions, and I'll start editing do the #867 manually.

**Pull request closed without merging**